Tests/TestingTests/RunnerTests.swift

```diff
@@ -501,7 +501,7 @@ final class RunnerTests: XCTestCase {
 
   func testPoundIfTrueTestFunctionRuns() async throws {
     let testStarted = expectation(description: "Test started")
-    testStarted.expectedFulfillmentCount = 4
+    testStarted.expectedFulfillmentCount = 5
```

Comment:
These increments are because a synthesized suite for `RunnerTests` is now included in the runner plan?

Reply:
Yes, exactly. Many of the fixture suites in this file are nested right within the `RunnerTests` XCTestCase subclass.